internal/api/ui/login/static/templates/change_password.html

```diff
@@ -4,7 +4,11 @@
     <h1>{{t "PasswordChange.Title"}}</h1>
     {{ template "user-profile" . }}
 
+    {{if .Expired}}
+    <p>{{t "PasswordChange.ExpiredDescription"}}</p>
```

Comment:
There seems to be a missing translation for this key:

![image](https://github.com/zitadel/zitadel/assets/5411563/ae033b16-2b00-483d-b8a6-2085c7de2304)